tests/unit/phpunit.xml

```diff
@@ -1,32 +1,22 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-<phpunit backupGlobals="false"
-         backupStaticAttributes="false"
-         colors="true"
-         convertErrorsToExceptions="true"
-         convertNoticesToExceptions="true"
-         convertWarningsToExceptions="true"
-         processIsolation="false"
-         stopOnFailure="false"
-         syntaxCheck="true"
-         bootstrap="./autoload.php">
-    <testsuite name="Boltpay_Unit_Test">
-        <directory>testsuite</directory>
-    </testsuite>
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="bootstrap.php" colors="true" backupGlobals="true">
+    <testsuites>
+        <testsuite name="Tests location">
```

Comment:
Is `Tests location` a more meaningful name than `Bolt_Unit_Test`?  It does not seem as descriptive to me.

Reply:
@mccottry This is just a label, not class or namespace

Reply:
Can you make the label be more descriptive like "Bolt_Unit_Test" or "Bolt M1 Plugin Unit Test"?  That label seems to more quickly and clearly explain what is going on to any reader of the code.